extensions/amp-pinterest/0.1/amp-pinterest.js

```diff
@@ -350,7 +350,7 @@ class AmpPinterest extends AMP.BaseElement {
         '?pin_ids=' + pinId +
         '&sub=www&base_scheme=https';
       return call(query).then(r => {
```

Comment:
@erwinmombay sorry, but I can't make either of these suggestions work. Help would be welcome, thanks!